Binaries/bin-conf/flhook_plugins/laz_NoDock.cfg

```diff
@@ -0,0 +1,973 @@
+nodock = EW17_04
+nodock = EW17_07
+nodock = GA01_10
+nodock = GA01_13
+nodock = GA02_03
+nodock = GA03_03
+nodock = GA09_02
+nodock = GA04_06
+nodock = GA07_06
+nodock = GA08_02
+nodock = BR10_02
+nodock = Bw10_05
+nodock = Br04_02
+nodock = GA06_10
+nodock = Bw08_02
+nodock = EW13_06
+nodock = EW15_02
+nodock = GA02_07
+nodock = GA05_02
+nodock = GA03_05
+nodock = GA04_03
+nodock = GA07_07
+nodock = GA08_03
+
+
+[GRN]
+ID = dsy_license_ga_n_grp
+nodock = Br06_01
+nodock = Br04_01_dock_ring
+nodock = Br04_01_docking_fixture
+nodock = Br06_04
+nodock = BR08_04
+nodock = BR08_01
+nodock = BR08_05
+nodock = EW13_07
+nodock = Bw08_04
+nodock = GA05_07
+nodock = Br06_02
+nodock = EW13_02
+nodock = EW13_03
+nodock = EW14_05
+nodock = GA06_08
+nodock = EW15_04
+nodock = EW15_09
+nodock = EW15_10
+nodock = EW17_03
+nodock = EW17_04
+nodock = EW17_07
+nodock = GA01_10
+nodock = GA01_13
+nodock = GA02_03
+nodock = GA03_03
+nodock = GA09_02
+nodock = GA04_06
+nodock = GA07_06
+nodock = GA08_02
+nodock = BR10_02
+nodock = Bw10_05
+
+[GRI]
+ID = dsy_license_ga_sol_grp
+nodock = ga03_11
+nodock = ga01_18
+nodock = ew17_05
+nodock = br04_05
+nodock = ew14_06
+
+[IDF]
+ID = dsy_license_ga_idf_grp
+nodock = Br04_04
+nodock = GA01_08
+nodock = GA02_06
+nodock = GA03_06
+nodock = GA04_04
+nodock = GA04_07
+nodock = GA08_06
+nodock = GA06_04
+
+[EFL]
+ID = dsy_license_ga_efl_grp
+nodock = Bw10_01_dock
+nodock = Bw10_docking_fixture_1
+nodock = EW14_03
+nodock = EW15_08
+nodock = GA01_04
+nodock = GA01_Roid_Miner_1
+nodock = GA02_05
+nodock = GA02_09
+nodock = GA05_05
+nodock = GA03_09
+nodock = GA04_10
+nodock = GA07_02
+nodock = GA07_03
+
+[GMS]
+ID = dsy_license_ga_gms_grp
+nodock = Br04_02
+nodock = GA06_10
+nodock = Bw08_02
+nodock = EW13_06
+nodock = EW15_02
+nodock = GA02_07
+nodock = GA05_02
+nodock = GA03_05
+nodock = GA04_03
+nodock = GA07_07
+nodock = GA08_03
+
+[GMG]
+ID = dsy_license_gd_gm_grp
+nodock = Bw05_01
+nodock = Bw05_02
+nodock = GA09_01
+nodock = Bw07_02
+nodock = BW69_02
+nodock = Ku04_05
+nodock = KU13_04
+nodock = KU13_06_dock_ring
+nodock = KU13_06_fixture
+nodock = BW69_03
+
+[CR]
+ID = dsy_license_colonialremnant
+nodock = bw14_03
+nodock = ga06_03
+nodock = iw11_03
+nodock = bw16_01
+nodock = bw16_04
+nodock = iw11_04_dock_ring
+nodock = iw11_04_docking_fixture
+nodock = iw11_05_dock_ring
+nodock = iw11_05_docking_fixture
+
+[The Core]
+ID = dsy_license_gd_bh_guardian
+nodock = EW18_02
+nodock = BW71_01
+nodock = GA01_16
+nodock = BW71_02
+nodock = BW71_03
+nodock = ST04_01
+
+[Corsairs]
+ID = dsy_license_fc_c_grp
+nodock = EW07_02
+nodock = Bw02_01
+nodock = Bw05_03
+nodock = HI08_01
+nodock = Ew03_01
+nodock = Ew06_05
+nodock = EW07_01
+nodock = EW19_03
+nodock = Hi02_dock_ring_1
+nodock = Hi02_docking_fixture_1
+nodock = Hi02_02
+nodock = Hi02_03
+nodock = HI18_01
+nodock = HI18_02
+nodock = HI18_03
+nodock = HI20_01
+nodock = HI20_02
+nodock = HI20_03
+nodock = HI20_04
+nodock = RH11_01
+nodock = IW13_08
+;Guard Stations
+HI18_04
```

Comment:
?

Reply:
I was initially going to outright restrict guard stations so I left them out, but then I realised that might be a bad idea for RP, but forgot to go and add the nodock comments.

Reply:
Those are keys but okay